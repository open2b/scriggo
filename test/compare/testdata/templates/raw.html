{# render #}
a
{% raw %}
b
{% end %}
c
{% raw %}d{% end raw %}
e
{% raw %}
{% if f %}{{ g }}{% end if %}
{% end %}
h
{% raw code %}
{% for i := 0; i < 5; i++ %}{{ i }}{% end if %}
{% end raw code %}
l
{% raw code %}
{% raw %}{{ m }}{% end row %}
{% end raw code %}
n
{% raw code %}
o {% raw code %}
{% end raw code %}
p
