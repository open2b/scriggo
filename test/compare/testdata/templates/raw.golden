
a
b
c
d
e
{% if f %}{{ g }}{% end if %}
h
{% for i := 0; i < 5; i++ %}{{ i }}{% end if %}
l
{% raw %}{{ m }}{% end row %}
n
o {% raw code %}
p
